include/nccl_ofi_tuner.h

```diff
@@ -16,6 +16,7 @@ struct nccl_ofi_tuner_model_params {
 	float internode_bw;
 	float intranode_bw;
 	int num_rails;
+	unsigned long long nccl_buffsize;
```

Comment:
reason this isn't a size_t?

Reply:
I suspected conversion/overflow issues at one point and its usage here and elsewhere in this file is this is an artifact of that. I can replace with uint64_ts